packages/neos-ui-redux-store/src/CR/Nodes/index.js

```diff
@@ -41,13 +49,41 @@ const focus = createAction(FOCUS, (contextPath, fusionPath) => ({contextPath, fu
  */
 const unFocus = createAction(UNFOCUS);
 
+/**
+ * Start node removal workflow
```

Comment:
Maybe Sagas could be a thing to orchestrate these smaller events together for certain workflows?